app/src/main/java/sopt/org/millie/presentation/search/book/SearchBookScreen.kt

```diff
@@ -0,0 +1,122 @@
+package sopt.org.millie.presentation.search.book
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.presentation.search.component.SearchTitle
+import sopt.org.millie.presentation.search.component.book.SearchBanner
+import sopt.org.millie.presentation.search.component.book.SearchBookItem
+import sopt.org.millie.presentation.search.model.SearchBannerModel
+import sopt.org.millie.presentation.search.model.SearchBookModel
+
+@Composable
+fun SearchBookScreen(
+    bookList: List<SearchBookModel>,
+    searchBanner: SearchBannerModel,
+    onClickBookItem: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .padding(vertical = 23.dp),
+    ) {
+        SearchTitle(
+            title = "도서",
+            count = bookList.size,
+            modifier = Modifier.padding(start = 17.dp),
+        )
+
+        Spacer(modifier = Modifier.height(10.dp))
+
+        LazyRow(
+            modifier = Modifier,
+            contentPadding = PaddingValues(horizontal = 17.dp),
+            horizontalArrangement = Arrangement.spacedBy(8.dp),
+        ) {
+            items(
+                count = bookList.size,
+            ) {
+                SearchBookItem(
+                    imageUrl = bookList[it].imageUrl,
+                    reader = bookList[it].reader,
+                    title = bookList[it].title,
+                    author = bookList[it].author,
+                    rate = bookList[it].rate,
+                    minute = bookList[it].minute,
+                    onClickBookItem = onClickBookItem,
+                    isAudioEnabled = bookList[it].isAudioEnabled,
+                )
+            }
```

Comment:
```suggestion
items(
                items = bookList,
                key = { it.bookId },
            ) { book ->
                SearchBookItem(
                    imageUrl = book.imageUrl,
                    reader = book.reader,
                    title = book.title,
                    author = book.author,
                    rate = book.rate,
                    minute = book.minute,
                    onClickBookItem = onClickBookItem,
                    isAudioEnabled = book.isAudioEnabled,
                )
            }
```
이거 사이즈로 받아온 이유가 있나용 ?!? 리스트 자체를 넘겨주는 방법도 있답니다 ㅎㅎ 그리고 아래에 Model에 bookId 값도 추가해서 key를 사용해주면 좋을 것 같아요 ~~ key의 장점은 심콩스 때도 한번 설명한 적 잇죠 ?!?! ㅎㅎㅎ

Reply:
리스트로 받고 싶었는데 뭔가 계속 오류가 떠서 일단 저렇게 해뒀었어요ㅜ 수정해야겠네요!!
key의 장점은.. key를 기준으로 아이템을 식별해서 변하지 않은 아이템의 리컴포지션을 막아준다.. 맞나용?

Reply:
요거 임포트문 추가가 안대서 그런가 ?? `import androidx.compose.foundation.lazy.items` 요거 추가해 보시와용 ~~

키의 장점 ㅎㅎ 마자요!!